dbt/clients/jinja.py

```diff
@@ -71,6 +71,8 @@ def __call__(self, *args, **kwargs):
             if path not in self.node['depends_on']['macros']:
                 self.node['depends_on']['macros'].append(path)
 
+            return True
```

Comment:
@cmcarthur curious what you think about this. By returning True instead of None here, dbt will evaluate the True branch of the conditional in the parsing stage. That means that we wouldn't throw an error for typical use cases like:

```sql
{% if already_exists(this.schema, this.table) %}
  select max(created_at) from {{ ref('other_model') }}
{% endif %}
```

The error would however occur if there was a `ref` in the `else` branch. Kind of a hack, but it feels a lot better to me. Could go either way here.